dozer-cli/src/live/watcher.rs

```diff
@@ -22,7 +23,13 @@ pub async fn watch(
 
     let watcher = debouncer.watcher();
 
-    watcher.watch(dir.as_path(), RecursiveMode::Recursive)?;
+    watcher.watch(dir.as_path(), RecursiveMode::NonRecursive)?;
```

Comment:
This will work against `dozer init`'s directory structure.

I remember we've already modified `home_dir` to a temporary directory. Maybe we only need to set `cache_dir` to a temporary directory too.